lib/commentOnOpenedPullRequest.js

```diff
@@ -93,6 +102,32 @@ commentOnOpenedPullRequest._askAboutChanges = function (files, baseBranch) {
     return true;
 };
 
+commentOnOpenedPullRequest._askAboutContributors = function (userName, repositorySettings) {
+    if (!defined(repositorySettings.contributorsUrl)) {
+        return Promise.resolve(false);
+    }
+
+    return requestPromise.get({
+        url: repositorySettings.contributorsUrl,
+        headers: repositorySettings.headers,
+        json: true
+    })
+    .then(function (response) {
+        var contributorsFileContent = Buffer.from(response.content, 'base64').toString();
+        // Matches items that look like: [<string>](https://github.com/<username>)
+        // sets the username as the first capturing group.
+        var regex = /\[.+\]\(https:\/\/github.com\/(.*)\)/g;
+        var match = regex.exec(contributorsFileContent);
+        while (defined(match)) {
```

Comment:
Actually, we already have their username, so can we just check if it matches any of the usernames in this file (rather than matching every username, then checking each)?

Reply:
That's a good idea!